### PR TITLE
Added JARGON SDK for Localization

### PR DESCRIPTION
**Pull request by TheDreamSaver**
Added the JARGON SDK for better localization and content management. All the output responses are now in a separate file.

### CODE REVIEW

**Review by acerbisgianluca — APPROVED** [review]
Perfect, I'm starting working on italian version

File: lambda/custom/helperFunctions.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-
 const axios = require('axios');
 const apiEndpoints = require('./apiEndpoints');
 const envVariables = require('./config');
 
-
+const Jargon = require('@jargon/alexa-skill-sdk')
+const ri = Jargon.ri
```
Comment (acerbisgianluca) [inline lambda/custom/helperFunctions.js:6]: Just a question. In this file, how can JARGON know what language it has to use?
Does it share the same instance from index.js where jargon inject its properties into the ask skill builder?

File: lambda/custom/index.js [hunk 2]
```diff
@@ -72,10 +75,10 @@ const DeleteChannelIntentHandler = {
       const headers = await helperFunctions.login(accessToken);
       const speechText = await helperFunctions.deleteChannel(channelName, headers);
 
-      return handlerInput.responseBuilder
+      return handlerInput.jrb
         .speak(speechText)
-        .withSimpleCard('Delete A Channel', speechText)
         .reprompt(speechText)
+        .withSimpleCard('DELETE_CHANNEL.CARD_TITLE', speechText)
```
Comment (acerbisgianluca) [inline lambda/custom/index.js]: Here you missed a call to ri()

**Review by acerbisgianluca — CHANGES_REQUESTED** [review]
Missed call to ri()

File: lambda/custom/index.js (hunk 2 above)
Comment (TheDreamSaver) [reply lambda/custom/index.js]: Thanks have put it in.

File: lambda/custom/helperFunctions.js (hunk 1 above)
Comment (TheDreamSaver) [reply lambda/custom/helperFunctions.js:6]: That's a really good question.
And I am not completely sure of the answer, but I have messaged Jargon team regarding this and we should find out soon.

**Review by acerbisgianluca — APPROVED** [review]
Now seems all good

**Comment by PrajvalRaval** [comment]
@TheDreamSaver I am having a little trouble with the speechtext, It is not returning as it was supposed to. I have attached a screenshot of the output I am getting.
Screenshot 2019-04-28 at 9 56 03 PM

**Comment by acerbisgianluca** [comment]
> @TheDreamSaver I am having a little trouble with the speechtext, It is not returning as it was supposed to. I have attached a screenshot of the output I am getting.
> Screenshot 2019-04-28 at 9 56 03 PM

You should use English (US) not English (IN). I think Jargon doesn't find any resource named en_IN

**Comment by PrajvalRaval** [comment]
> > @TheDreamSaver I am having a little trouble with the speechtext, It is not returning as it was supposed to. I have attached a screenshot of the output I am getting.
> > Screenshot 2019-04-28 at 9 56 03 PM
> 
> You should use English (US) not English (IN). I think Jargon doesn't find any resource named en_IN

Ok That Worked Out but wouldn't that be an issue. Do we have to create a resource file for every English language model that we are going to use in our skill ?

**Comment by TheDreamSaver** [comment]
@PrajvalRaval No that isn't an issue. We have to add it separately for each different locale. That's how localization works. Also remember we have a separate interaction model for each different locale.

Some things in English US are different for English India. Same goes for other three English locales as well.

**Comment by PrajvalRaval** [comment]
> Some things in English US are different English India. Same goes for other three English locales as well.

Great No issues on my side then for this PR. Still have to see how to integrate Non-English Language 👍

**Review by PrajvalRaval — APPROVED** [review]

**Comment by acerbisgianluca** [comment]
> > Some things in English US are different English India. Same goes for other three English locales as well.
> 
> Great No issues on my side then for this PR. Still have to see how to integrate Non-English Language 👍

You need to create a new file inside `resources` with the same structure as en_US, a new file inside `models` with the same structure as en_US and a new block inside `skill.json`. I will make my PR with italian language soon.
Please note that FallbackIntent only exists in some languages. For example, in italian and spanish ones it doesn't exist.

File: lambda/custom/helperFunctions.js (hunk 1 above)
Comment (acerbisgianluca) [reply lambda/custom/helperFunctions.js:6]: BTW I tested it using italian and creating a new channel with 2 words (for obtaining an error). It worked anyways so I think they share the same instance allowing them to know what language is in use.

Comment (TheDreamSaver) [reply lambda/custom/helperFunctions.js:6]: Awesome. Then we can merge this one @Sing-Li

**Comment by TheDreamSaver** [comment]
To understand how Jargon localization works with multiple files check https://github.com/JargonInc/jargon-sdk-nodejs/issues/4

**Pull request merged**